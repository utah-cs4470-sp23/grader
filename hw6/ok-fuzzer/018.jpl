show (- ((197 / (- 713)) % (if (! false) then (472 - (- 75)) else ((- 649) / (459 % 933)))))

