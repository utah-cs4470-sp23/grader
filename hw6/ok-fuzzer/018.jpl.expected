(ShowCmd
 (ArrayLiteralExpr
  (ArrayType (BoolType) 1)
  (BinopExpr
   (BoolType)
   (BinopExpr
    (FloatType)
    (UnopExpr (FloatType) - (FloatExpr (FloatType) 51))
    %
    (FloatExpr (FloatType) 36))
   !=
   (UnopExpr
    (FloatType)
    -
    (BinopExpr
     (FloatType)
     (IfExpr
      (FloatType)
      (TrueExpr)
      (FloatExpr (FloatType) 12)
      (FloatExpr (FloatType) 35))
     +
     (FloatExpr (FloatType) 30))))))
